app/backend/wells/views.py

```diff
@@ -69,7 +69,7 @@ class WellDetail(RetrieveAPIView):
     """
     serializer_class = WellDetailSerializer
 
-    queryset = Well.objects.all()
+    queryset = Well.objects.exclude(well_publication_status='Unpublished')
```

Comment:
A user editing a well still needs this endpoint to work - so either we have to no filter here, or only filter on users who don't have edit permissions. This endpoint is used in app/frontend/src/submissions/views/SubmissionsHome.vue for loading the form.